radio/src/Makefile

```diff
@@ -784,6 +784,12 @@ ifeq ($(PCB), $(filter $(PCB), SKY9X 9XRPRO AR9X))
     FLAVOUR = 9xrpro
     CPPDEFS += -DREVX
     LDSCRIPT = targets/sky9x/sam3s8c_flash.ld
+  ifeq ($(NANO), YES)
+    # use newlib-nano for linking
+    NEWLIB_NANO_FLAGS = --specs=nano.specs -u _printf_float
+    CPPDEFS += -DNANO
+  endif
+
```

Comment:
I think it's not a bad idea to have it available on 9XR-PRO, but I wonder if it's possible to avoid these 4-5 lines in all ARM platforms, perhaps it would be good to write this line elsewhere, no? By the way on 9XR-PRO malloc is only used for printf if I am not mistaken. The debate NANO/No NANO comes when we speak about Lua scripts ...